### DISCUSSION TITLE
Is it possible that kor give result for one pdf and does not give result for second same format. ?

### DISCUSSION THREAD

**Discussion by shividwivedi** [eyurtsev/kor#289] [Q&A] [answered]
I have a pdf with exporter XYZ kor gives correct result for that but when i tried another pdf of same exporter XYZ with same format it is not parsing anything.

**Comment by eyurtsev** [comment, marked as answer]
Kor is just a wrapper around to LLMs to make it easier to do extraction with reference examples. LLMs are far from perfect. Follow the guidelines here https://eyurtsev.github.io/kor/guidelines.html
